addons/macrochanger/data/settings.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.1" ?>
```

Comment:
You should remove this file, it will be auto-generated on the first run with the default settings. If you ever update the default settings, this file will not be updated and any new settings will not be copied over. Instead the config library's own mechanism will add new default keys automatically, so you should just use the existing mechanism.